pkg/projects/Microsoft.NETCore/Microsoft.NETCore.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>5.0.3</Version>
+    <Version>5.0.2</Version>
```

Comment:
Should I temporarily remove my change to pkg/stable.packages.props to make sure we don't produce a Microsoft.NETCore? Or is that unnecessary?